hoss/exceptions.py

```diff
@@ -108,6 +108,36 @@ def __init__(self):
         )
 
 
+class MissingCoordinateDataset(CustomError):
```

Comment:
First up - nice extension of the `Exception`s in this file.

But, it's not really clear what the difference between these two new exceptions is. The description is the same for both. Do you need them both?

Reply:
One is if one of the datasets are missing. The MissingValidCoordinateDataset datasets is raised when the datasets have a lot of fill values and it cannot be used to determine the GeoCorners.. If the corners have fill values, the method throws an exception and does not move to the next row/col on the grid. DAS-2236. David had some notes on handling those cases

Reply:
updated the comments - https://github.com/nasa/harmony-opendap-subsetter/pull/15/commits/f628166ed1d8bc3d1e879ba73332c1475cacf728

Reply:
I think the naming of the exception could perhaps be improved then - the coordinates variables aren't missing. Perhaps just `InvalidCoordinateVariable` (also I think "Variable" rather than "Dataset" is preferred in this code to keep language consistent).